src/routes/api/idea.js

```diff
@@ -236,6 +236,25 @@ function createIdeaJSON(idea, user) {
 	result.config = null;
 	result.site = null;
 	result.can = can;
+
+
+	// Fixme: hide email in arguments
+  if(idea.argumentsAgainst) {
+    result.argumentsAgainst = result.argumentsAgainst.map((argument) => {
+      argument.user.email = user.role === 'admin' ? argument.user.email : '';
+
+      return argument;
+    });
+  }
+
+  if(idea.argumentsFor) {
+    result.argumentsFor = result.argumentsFor.map((argument) => {
+    	argument.user.email = user.role === 'admin' ? argument.user.email : '';
+
+    	return argument;
+		});
+  }
```

Comment:
Hebben deze argumenten ook nog onderliggende `reactions`? En wordt daar het e-mail adres ook uitgehaald?
Wordt hier bij `idea` ook de `phone` uit de `extraData` gehaald?

Reply:
hmm, deze commit hoort er niet bij. dit is van de hotfix